http/http_connection.hpp

```diff
@@ -395,10 +397,9 @@ class Connection :
             handler->handleUpgrade(*req, res, std::move(adaptor));
             // delete lambda with self shared_ptr
             // to enable connection destruction
-            res.completeRequestHandler = nullptr;
+            asyncResp->res.setCompleteRequestHandler(nullptr);
```

Comment:
This is an example of the change I'm commenting on line 179 of http_response.hpp. 
Have we made all of these necessary changes?

Reply:
> Have we made all of these necessary changes?

Yes, we did.